cbw_api_toolbox/cbw_api.py

```diff
@@ -67,11 +74,15 @@ def ping(self):
 
     def servers(self):
         """GET request to /api/v2/servers to get all servers"""
-        return [CBWParser().parse(CBWServer, server) for server in self._request(ROUTE_GET_SERVERS)]
+        return [CBWParser().parse(CBWServer, server) for server in self._request(
```

Comment:
Why 3 lines ?

Reply:
Pylint doesn't want to let one line, too many characters ... (Pylint message: "Line too long 122/100")

Reply:
Mettre 

```python
  for server in self._request("GET", [ROUTE_SERVERS])] : 
     CBWParser().parse(CBWServer, server)
```